pkgs/servers/sql/postgresql/libpq.nix

```diff
@@ -0,0 +1,70 @@
+{ stdenv
+, lib
+, openssl
+, zlib
+, libkrb5
+, icu
+, postgresql
+, pkg-config
+, tzdata
+, gssSupport ? with stdenv.hostPlatform; !isWindows && !isStatic
+}:
+
+stdenv.mkDerivation {
+  pname = "libpq";
+  version = postgresql.version;
+  src = postgresql.src;
+
+  configureFlags = [
+    "--with-openssl"
+    "--with-icu"
+    "--without-readline"
+    "--with-system-tzdata=${tzdata}/share/zoneinfo"
+  ]
+  ++ lib.optionals gssSupport [ "--with-gssapi" ];
+
+  nativeBuildInputs = [ pkg-config tzdata ];
+  buildInputs = [ openssl zlib icu ]
+    ++ lib.optional gssSupport libkrb5;
+
+  buildPhase = ''
+    runHook preBuild
+
+    make -C src/bin/pg_config
+    make -C src/common
+    make -C src/include
+    make -C src/interfaces/libpq
+    make -C src/port
```

Comment:
This is ignoring buildFlags and other standard flags like -j$NIX_BUILD_CORES

Reply:
Is there an idiomatic way of doing that when overriding the `buildPhase`? Or should I just include `$buildFlags`?

Reply:
I don't think so. Please note that `SHELL=$SHELL` is also relevant. Not sure if we want to include `makeFlags`/etc in each invocation here as well.

Regarding what's exactly passed to `make` in `buildPhase` by default, please look at the `buildPhase`-function in `pkgs/stdenv/generic/setup.sh`.

Reply:
Turns out there's a `submake-libpq` makefile target, so we can sidestep this entire issue.